src/core/connection.c

```diff
@@ -5820,13 +5820,8 @@ QuicConnProcessRouteCompletion(
     QUIC_PATH* Path = QuicConnGetPathByID(Connection, PathId, &PathIndex);
     if (Path != NULL) {
         if (Succeeded) {
-            QuicTraceLogConnInfo(
```

Comment:
FYI, I removed this trace because it's redundant with the one logged by CxPlatResolveRouteComplete.

Reply:
Sidecar

Reply:
@maolson-msft does the other event log the path ID, like it does here?

Reply:
Yep, it does.